### PR TITLE
Fix minor typos in documentations, comments and test descriptions

### PR DESCRIPTION
**Pull request by marcwrobel**

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const daysInWeek = 7
  * Days in 1 year
  * One years equals 365.2425 days according to the formula:
  *
- * > Leap year occures every 4 years, except for years that are divisable by 100 and not divisable by 400.
+ * > Leap year occurs every 4 years, except for years that are dividable by 100 and not dividable by 400.
```
Comment (fturmel) [inline src/constants/index.ts:15]: "divisible" (over "dividable") _might_ be the more common and accurate word to use in this context.
https://english.stackexchange.com/questions/85111/when-to-use-divisible-vs-dividable

**Comment by fturmel** [comment]
Nice! Pretty sure the contributing.md ones were mine 🤦

File: src/constants/index.ts (hunk 1 above)
Comment (marcwrobel) [reply src/constants/index.ts:15]: I think you are right, I will revert those changes.

Comment (marcwrobel) [reply src/constants/index.ts]: Done.

File: docs/release.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 1. First, make sure that the library is built by running `./scripts/build/build.sh` and committing and pushing any change you would have.
 
-2. Then add the changelog entry generated by `yarn ts-node scripts/release/buildChangelog.ts` to (CHANGELOG.md)[../CHANGELOG.md]. Make sure that the output is valid Markdown and fix if there're any errors. Commit and push the file.
+2. Then add the changelog entry generated by `yarn ts-node scripts/release/buildChangelog.ts` to (CHANGELOG.md)[../CHANGELOG.md]. Make sure that the output is valid Markdown and fix if there is any error. Commit and push the file.
```
Comment (fturmel) [inline docs/release.md:5]: "there're" seems like an unusual contraction to me, but not wrong per se.

How about simply: "Make sure that the output is valid Markdown and fix any errors."?

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 3]
```diff
@@ -25,4 +25,4 @@ fn()
 - date-fns version: <!-- Fill in the date-fns version you use -->
 - Browser/Node.js version: <!-- Fill in the envrionment version (i.e. Chrome 94.0.4606.61) -->
 - Your timezone: <!-- Fill in your timezone name (i.e. Asia/Singapore) -->
-- Your current time: <!-- Fill in the time when did you tested the problem (i.e. 16:20) -->
+- Your current time: <!-- Fill in the time when did you test the problem (i.e. 16:20) -->
```
Comment (fturmel) [inline .github/ISSUE_TEMPLATE/issue-report.md:28]: "Fill in the local time at which you tested the problem" would be better imo.

Per #2831 this issue template has never been functional, so it's somewhat inconsequential.

**Comment by fturmel** [comment]
Most of these are in unit tests, but I'll still tag this PR as documentation improvements.

File: docs/release.md (hunk 2 above)
Comment (marcwrobel) [reply docs/release.md]: You are right, this is better. Sentence updated.

File: .github/ISSUE_TEMPLATE/issue-report.md (hunk 3 above)
Comment (marcwrobel) [reply .github/ISSUE_TEMPLATE/issue-report.md]: You are right, this is better. Sentence updated.

**Review by fturmel — APPROVED** [review]
Great!

**Comment by marcwrobel** [comment]
@fturmel, are you authorized to approve workflows and merge this branch ?

**Comment by fturmel** [comment]
@marcwrobel no write permissions on the main branch

**Comment by marcwrobel** [comment]
Ok, thanks for the info. Hope there will be no more conflicts until it's merged 🤞.

**Comment by marcwrobel** [comment]
@tan75, could you have a look at this PR and let me know if there is a chance it can be merged soon ?

**Comment by marcwrobel** [comment]
This PR now have conflicts with the main branch. Should I resolve those conflicts or close it ?

**Comment by marcwrobel** [comment]
It's been almost a year now since I opened this PR. Closing it as it should be out of date now.

**Pull request closed without merging**